cnapy/appdata.py

```diff
@@ -21,7 +21,8 @@
 class AppData:
     ''' The application data '''
 
-    def __init__(self):
+    def __init__(self, qapp):
+        self.qapp = qapp
         self.version = "cnapy-1.1.1"
         self.format_version = 1
```

Comment:
Would you also have to increase the format version?

Reply:
Good point, will set
self.format_version = 2